### ISSUE TITLE
Hi~I wondering the impact of loss function on segmentation accuracy.

### ISSUE THREAD

**Issue by CyberPegasus** [hanchaoleng/ShapeConv#15]
Have the authors tried other combination of loss functions, e.g., CE + DICE and Focal + DICE, rather than CrossEntropy (CE) ? 
Does the simple application of CrossEntropy (CE) in ShapeConv's implementation achieve the nearly optimal performance ?
